distribution.pm

```diff
@@ -116,10 +116,11 @@ Deprecated mode
 Run I<$cmd> (by assuming the console prompt and typing the command). After that, echo
 hashed command to serial line and wait for it in order to detect execution is finished.
 To avoid waiting, use I<$timeout> 0.
+In case you have mistyping problems lower I<$type_speed> from its default value of 250
```

Comment:
```suggestion
 C<$type_speed> is 250 by default, lower this in case you have typing issues 
```

Reply:
I tried to be consistent with the other parameters